cypress/e2e/awx/resources/inventoriesConstructed.cy.ts

```diff
@@ -0,0 +1,197 @@
+      cy.createAwxInventory({ organization: organization.id }).then((inv) => {
+        inventoriesList.push(inv);
+      });
+    }
+    cy.createAwxConstructedInventory(organization).then((constInv) => {
+      constructedInv = constInv;
+    });
+  });
+
+  afterEach(() => {
+    constrInvToDelete.push(constructedInv);
+    invToDelete.push(...inventoriesList);
+  });
+
+  after(() => {
+    cy.deleteAwxInstanceGroup(instanceGroup);
+    invToDelete.map((inventory) => cy.deleteAwxInventory(inventory, { failOnStatusCode: false }));
+    constrInvToDelete.map((constrInventory) => cy.deleteAwxConstructedInventory(constrInventory));
+    cy.deleteAwxOrganization(organization);
+  });
+
+  it('can create a constructed inventory using specific source_vars and limit and then delete that inventory', () => {
+    invNames = inventoriesList.map(({ name }) => String(name));
+    cy.intercept('POST', awxAPI`/constructed_inventories/`).as('createInv');
+    const constInvName = 'E2E Constructed Inventory ' + randomString(4);
+    // generates random values to be used during the test.
+    const cacheTimeoutValue = generateRandom(0, 15);
+    // which combination should be used to test verbosity from 1 to 5 - UI forces to use only 0-2
+    const verbosityValue = generateRandom(0, 2);
+
+    cy.navigateTo('awx', 'inventories');
+    cy.getByDataCy('create-inventory').click();
+    cy.get('.pf-v5-c-dropdown__menu').within(() => {
+      cy.get('[data-cy="create-constructed-inventory"]').click();
+    });
+    cy.getByDataCy('name').type(constInvName);
+    cy.getByDataCy('description').type(`Description of "${constInvName}" typed by Cypress`);
+    cy.intercept({
+      method: 'GET',
+      pathname: awxAPI`/organizations/`,
+      query: { name__icontains: organization.name },
+    }).as('filterOrg');
+    cy.singleSelectBy('[data-cy="organization"]', organization.name);
+    cy.wait('@filterOrg');
+    // this can be simplified if we include data-cy to the search button of instance groups
+    cy.getByDataCy('instance-group-select-form-group').find('[aria-label="Options menu"]').click();
+    cy.intercept({
+      method: 'GET',
+      pathname: awxAPI`/instance_groups/`,
+      query: { name: instanceGroup.name },
+    }).as('filterInstanceG');
+    cy.filterTableByMultiSelect('name', [instanceGroup.name]);
+    cy.wait('@filterInstanceG');
+    cy.getByDataCy('checkbox-column-cell').click();
+    cy.clickModalButton('Confirm');
+    cy.intercept({
+      method: 'GET',
+      pathname: awxAPI`/inventories/`,
+      query: { name__icontains: invNames[invToCreate - 1] },
+    }).as('filterInputInventories');
+    cy.multiSelectBy('[data-cy="inventories"]', invNames);
+    cy.wait('@filterInputInventories');
+    cy.getByDataCy('update_cache_timeout').clear().type(String(cacheTimeoutValue));
+    cy.singleSelectByDataCy('verbosity', String(verbosityValue));
+    // FIXME: when verbosity is 0 the filed remains empty
```

Comment:
Is this comment required?

Reply:
The issue was fixed. I'm removing the comment now.